README.md

```diff
@@ -985,7 +985,7 @@ Waste collection schedules in the following formats and countries are supported.
 - [Waldshut](/doc/source/app_abfallplus_de.md) / Abfall+ App: unterallgaeu
 - [WBO Wirtschaftsbetriebe Oberhausen](/doc/source/abfallnavi_de.md) / wbo-online.de
 - [Wegberg (MyMuell App)](/doc/source/jumomind_de.md) / mymuell.de
-- [Wermelskirchen](/doc/source/wermelskirchen_de.md) / wermelskirchen.de
+- [Wermelskirchen (Service Down)](/doc/source/wermelskirchen_de.md) / wermelskirchen.de
```

Comment:
Unsure why this happens but it is consistently down for me

Reply:
This is expected we renamed this source, but it seems like the update script was not run on the version you forked. It is already like this on the current master